src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -118,7 +118,14 @@ public function getConfigTreeBuilder()
                         ])
                     ->end()
                     ->arrayNode('types')
-                        ->prototype('scalar')->end()->defaultValue([
+                        ->prototype('array')
+                            ->children()
+                                ->enumNode('type')->values(['document', 'image', 'video', 'audio'])->isRequired()->end()
```

Comment:
This would dissallow that somebody adds a new type would not change this in a hotfix release

Reply:
Yes I agree, I made an update so it is possible now to add new types.